src/gameComponents/Ship/ProtoShip.java

```diff
@@ -0,0 +1,21 @@
+package gameComponents.Ship;
+
+import BBDGameLibrary.Geometry2d.BBDPoint;
+
+/**
+ * A class to serve as a precursor to a proper ship class.  Primarily serves as a testing
+ * bed for building new objects and the like.
+ */
+public class ProtoShip {
+    private enum Size {SMALL, MEDIUM, LARGE}
+
+    private BBDPoint currentLocation = new BBDPoint(0,0);
+    private float orientation = 0f;
```

Comment:
radians?

Reply:
degrees I think. Degrees are human readable and easier to write/comprehend

Reply:
Front facing BBDGeometry rotations are degrees, but use radians under the hood.

Reply:
cool. may be good to put is an inline comment